core/src/main/java/org/apache/carbondata/core/metadata/SegmentFileStore.java

```diff
@@ -184,6 +184,22 @@ public static String writeSegmentFile(CarbonTable carbonTable, String segmentId,
     return writeSegmentFile(carbonTable, segmentId, UUID, null, segPath);
   }
 
+  /**
+   * Returns the list of index files
+   *
+   * @param segmentPath
+   * @return
+   */
+  public static CarbonFile[] getListOfCarbonIndexFiles(String segmentPath) {
```

Comment:
Is the input parameter point to segment folder of transactional table?

Reply:
Yes, its the path to segment folder to get the carbon file.